modules/redpanda/redpanda_test.go

```diff
@@ -5,19 +5,34 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"io"
 	"net/http"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/network"
 	"github.com/twmb/franz-go/pkg/kadm"
 	"github.com/twmb/franz-go/pkg/kerr"
 	"github.com/twmb/franz-go/pkg/kgo"
 	"github.com/twmb/franz-go/pkg/sasl/scram"
 )
 
+// TestContainersLogger is a logger that implements the testcontainers
+// logger interface.
+type TestContainersLogger struct {
+	// LogPrefix is a string that is prefixed for every individual log line.
+	LogPrefix string
+}
+
+// Accept prints the log to stdout
+func (lc TestContainersLogger) Accept(l testcontainers.Log) {
+	fmt.Print(lc.LogPrefix + string(l.Content))
+}
+
```

Comment:
Is this used elsewhere?

Reply:
Good catch, this is a leftover I used for debugging, gonna remove it.